lib/hammer_cli_foreman_templates/import.rb

```diff
@@ -5,9 +5,46 @@ class ImportCommand < HammerCLIForeman::Command
 
     resource :template, :import
 
-    success_message _('Import finished')
+    success_message _('Import finished.')
     failure_message _('Could not import')
 
-    build_options without: [:verbose]
+    output do
+      field :id, _('Id'), Fields::Id
+      field :name, _('Name')
+      field :type, _('Type')
+      field :imported, _('Imported')
+      field :changed, _('Changed')
+      field nil, _('Changes'), Fields::Label, hide_blank: true do
+        field :diff, nil, Fields::LongText, hide_blank: true
+      end
+      field nil, _('Errors'), Fields::Label, hide_blank: true do
+        field :validation_errors, _('Validation'), Fields::Collection, hide_blank: true do
+          field nil, nil
+        end
+        field :additional_errors, _('Additional'), Fields::Collection, hide_blank: true do
+          field nil, nil
+        end
+        field :exception, _('Exception'), Fields::LongText, hide_blank: true
+      end
+    end
+
+    def execute
+      templates = send_request['templates']
+      if option_verbose
+        templates.each do |template|
+          if template['validation_errors'].empty?
+            template['validation_errors'] = nil
+          else
+            template['validation_errors'] = template['validation_errors'].values
```

Comment:
the validation errors in JSON look like this: `"validation_errors":{"name":"can't be blank"}` so I think you need to concat with keys as well to get more meaningful message, otherwise it is not entirely clear what attribute the message refers to:

![](https://i.imgur.com/H5a8cUn.png)